### ISSUE TITLE
Production cluster does not enable & start bacalhau service automatically

### ISSUE THREAD

**Issue by enricorotundo** [bacalhau-project/bacalhau#1671]
```
for x in {0..4}; do gcloud compute ssh "bacalhau-vm-production-$x"  --project "bacalhau-production" -- sudo systemctl enable bacalhau-daemon; done

for x in {0..4}; do gcloud compute ssh "bacalhau-vm-production-$x"  --project "bacalhau-production" -- sudo systemctl start bacalhau-daemon; done

gcloud compute ssh "bacalhau-vm-production-4" --zone europe-west4-a  --project "bacalhau-production" -- sudo systemctl enable bacalhau-daemon

gcloud compute ssh "bacalhau-vm-production-4" --zone europe-west4-a  --project "bacalhau-production" -- sudo systemctl start bacalhau-daemon
```

**Comment by wjam** [comment]
Fixed by changes in #1662?

**Comment by enricorotundo** [comment]
> Fixed by changes in #1662?

Don't think this has been fixed. It needs more investigation to pinpoint what the root cause is
